app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
Can we tell if an item is withdrawn? I think we care about this distinction in publishing apps and rummager.